### ISSUE TITLE
Better Naming

### ISSUE THREAD

**Issue by jeffturcotte** [imarc/clockvine-vue#2]
vuexModule, vueMixin, etc. are too generic IMO. I don't know for sure what would be better, but here are some suggestions:

ClockvineStore (capitalized as it is a class)
clockvineModel
clockvineEditableModel

**Comment by khamer** [comment]
I agree – I'm going to make 0.4 or 0.5 break backwards compatibility, and I'll do something like this then.

**Comment by khamer** [comment]
So, I did change the names, but didn't namespace them. It seems to me like

```
import { CollectionComponent } from 'clockvine-vue'
```

Is reasonable and consistent. If anything, I think I should try to publish as just 'clockvine'.
